app/services/feature.js

```diff
@@ -128,6 +128,9 @@ export default Service.extend({
         return this.lazyLoader.loadStyle('dark', 'assets/ghost-dark.css', true).then(() => {
             $('link[title=dark]').prop('disabled', !nightShift);
             $('link[title=light]').prop('disabled', nightShift);
+        }).catch((err) => {
```

Comment:
Is this gonna cause a lint issue, because of unused var? Maybe we should log it out?

Reply:
@allouis Yeah it will, good catch. I'll remove `err` for now and leave a comment